routing/geometry.cpp

```diff
@@ -51,14 +51,18 @@ namespace routing
 {
 // RoadGeometry ------------------------------------------------------------------------------------
 RoadGeometry::RoadGeometry(bool oneWay, double speed, Points const & points)
-  : m_points(points), m_speed(speed), m_isRoad(true), m_isOneWay(oneWay)
+  : m_points(points), m_speed(speed), m_isOneWay(oneWay)
```

Comment:
"ping on this comment" usually means that "I see the Review fixes commit, but can't find your reply or changes related to this comment".